package.json

```diff
@@ -98,9 +98,13 @@
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
     "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#ee14b3255da33d2b6924c357f43393251b6dc6d2",
+    "expo": "^49.0.0",
+    "expo-asset": "~8.10.1",
```

Comment:
Do we have to use `expo-asset`? I would prefer to only use `expo-image`. We are [working with Callstack to port react-native-asset directly into the React Native CLI](https://github.com/Expensify/App/issues/13772#issuecomment-1825545287), so that's what we'll prefer to use.